src/main/java/com/minecraftabnormals/abnormals_core/core/util/BlockUtil.java

```diff
@@ -118,4 +121,42 @@ public static BBRotation getRotationForDirection(Direction currentDirection, Dir
 			}
 		}
 	}
+
+	/**
+	 * <p>Gets the {@link BlockPos} in front of a dispenser.</p>
+	 * {@link BlockState#getBlock() source.getBlockState().getBlock()} must return an instance of {@link DispenserBlock}.
+	 *
+	 * @param source The {@link IBlockSource} to get the position from.
+	 * @return The position in front of the dispenser's output face.
+	 *
+	 * @author abigailfails
+	 * */
+	public static BlockPos dispenserOffsetPos(IBlockSource source) {
```

Comment:
I don't like this method being called `dispenserOffsetPos()` cause that is not what it does exactly.
In actuality, it just offsets the position in the direction of a state's `DirectionalBlock.FACING` property.

Reply:
Oh yeah, forgot that `DispenserBlock.FACING` is just `DirectionalBlock.FACING`. I'll update the names and documentation to reflect that

Reply:
Refactoring done in latest commit.